CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,24 @@
+# Code of Conduct
+
+The Twin Cities Mutual Aid Map community welcomes participation from everyone. 
+
+## Expectations 
+
+Our community expects that participants:
+
+* Create a community where members are safe, respected, friendly, and free to fully express their identities
+* Provide an environment where people are free from discrimination and harassment
+* Take responsibility for their words and actions
+* Value everyone’s ideas, skills, and contributions
+* Encourage participation, questions, and discussion
+* Don’t assume everyone has the same context 
+* Find ways for people to productively contribute their time and skills (technical or not) 
```

Comment:
What different ways can we welcome, encourage and acknowledge all contributions without having two buckets? I read `technical` here as `code` (and find it easy to read a hierarchy in that relationship) but one could argue that data analysis or any other range of skills are technical. OTC has been working to elevate outreach, process/system mapping, data gathering and analysis and other "non-technical" activities. I know this is a bullet from the CfA CoC (https://brigade.codeforamerica.org/about/code-of-conduct) but welcome ideas.

Reply:
?
> Value everyone’s ideas and find ways for people to productively contribute their time and skills

Reply:
I like that proposal!

Reply:
@grebbesen That's a great point. I agree on the hierarchy implication. I'll remove the qualification for now. I do wonder if the context of Github overly implies code and/or if qualification is necessary at all (I'm genuinely not sure). Happy to update with any ideas from others!